ca/ocsp.go

```diff
@@ -270,3 +270,33 @@ func (olq *ocspLogQueue) stop() {
 	close(olq.queue)
 	olq.wg.Wait()
 }
+
+// disabledOCSPImpl implements the capb.OCSPGeneratorServer interface, but
+// returns an error for all gRPC methods. This is only used to replace a real
+// impl when the OCSPGenerator service is disabled.
+// TODO(#6448): Remove this.
+type disabledOCSPImpl struct {
+	capb.UnimplementedOCSPGeneratorServer
+}
+
+func NewDisabledOCSPImpl() *disabledOCSPImpl {
+	return &disabledOCSPImpl{}
+}
+
+func (oi *disabledOCSPImpl) GenerateOCSP(ctx context.Context, req *capb.GenerateOCSPRequest) (*capb.OCSPResponse, error) {
+	return nil, errors.New("the OCSPGenerator gRPC service is disabled")
+}
+
+func (oi *disabledOCSPImpl) LogOCSPLoop() {}
```

Comment:
Potentially return an error like you do with `GenerateOCSP()` above. Otherwise, comment it as a noop. I'm partial to the former since we don't want anyone thinking that this works.

Reply:
This isn't a gRPC method that can be called remotely, this is an extra add-on that is only called by boulder-ca/main.go. It has to satisfy the same interface as the existing ocspImpl, which doesn't have any return value because all it does is start a goroutine. It could panic, I guess?

Reply:
Let's comment this as a noop then. I'm not sure why I thought you could modify the return value here.

Reply:
Okay I've added comments here and in crl.go more clearly describing why these methods are the way they are.